src/components/Contents.jsx

```diff
@@ -0,0 +1,40 @@
+import NavBar from "./NavBar";
+import Reviews from "./Reviews";
+import { getReviews } from "../api";
+import { useEffect, useState } from "react";
+import "../App.css";
+
+function Contents() {
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    getReviews().then((reviews) => {
+      setReviews(reviews);
+    });
+  }, []);
+
+  return (
```

Comment:
Would like to see some kind of loading state for the page whilst we fetch the data. This provides a better user experience so the user knows that something is happening and is more likely to wait on the page